docs/tr/docs/features.md

```diff
@@ -194,7 +193,7 @@ Aynı şekilde, databaseden gelen objeyi de **direkt olarak isteğe** de tamamiy
 
 * **Kafa karıştırmaz**:
     * Farklı bir syntax öğrenmenize gerek kalmaz,
```

Comment:
Bu satır sonunda virgül, sonraki satırların sonunda nokta kullanılmış.

Syntax yerine `sözdizimi` kullanıyoruz.

Aşağıda ki kısımlarda en güncel en/docs/index.md ile uyumlu değil, eşit değil.